Loops/Problem12/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Problem12
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int quotient = int.Parse(Console.ReadLine());
+
+            Stack<int> stack = new Stack<int>();
```

Comment:
Why do we need to use
Stack<int> stack = new Stack<int>(); String?

Reply:
Because the formula for conversion must have the numbers in opposite order